client/src/relay/updaters.tsx

```diff
@@ -103,6 +102,31 @@ export function createMessageUpdater(
   updateChannelAndRearrange(store, channelId);
 }
 
+/**
+ * Optimistically update Relay store after `createMessage` mutation.
+ * @param store Relay store proxy for updater.
+ * @param channelId ID of the channel to be updated.
+ * @param text Text content of the message.
+ * @param userId ID of the auth user.
+ */
+export function createMessageOptimisticUpdater(
+  store: RecordSourceSelectorProxy,
+  channelId: string,
+  text: string,
+  userId: string,
+): void {
+  const messageProxy = store.create(
+    `client:message:${Math.floor(Math.random() * 1000)}`,
```

Comment:
Can we have the better unique key here?

Reply:
Should we use something like nanoid here?

Reply:
Would be great!